### DISCUSSION TITLE
[Improvement] 2FA Input

### DISCUSSION THREAD

**Discussion by CodeWithDennis** [backstagephp/filament-2fa#27] [Ideas]
When you log in and see the 2FA field, the recovery codes field shows up too. It might be better if this was on a separate page from a clickable link. If that sounds like a good idea, it could also be nice to redesign the page a bit—maybe use six number slots for input instead of a single input box (just a thought).

**Comment by Baspa** [comment]
Discussed this a few days ago with a colleague as well. The recovery field shouldn't show up immediately. We thought about using the same input for the `recovery` code as well as the `otp` code. But maybe a separate page would be good as well.
